lib/phoenix.js

```diff
@@ -130,13 +103,14 @@ module.exports.getCampaignById = function getCampaignById(campaignId) {
     .catch(err => Promise.reject(err));
 };
 
+/**
+ * @param {Array} campaignIds
+ * @return {Promise}
+ */
 module.exports.fetchCampaigns = function (campaignIds) {
-  logger.debug(`phoenix.fetchCampaigns:${campaignIds}`);
-  const endpoint = `campaigns?ids=${campaignIds.join(',')}`;
-
-  return new Promise((resolve, reject) => {
-    executeGet(endpoint)
-      .then(res => resolve(res.data.map(campaign => exports.parsePhoenixCampaign(campaign))))
-      .catch(err => reject(exports.parsePhoenixError(err)));
+  const promises = [];
+  campaignIds.forEach((campaignId) => {
+    promises.push(module.exports.getCampaignById(campaignId));
```

Comment:
This looks a little cringe inducing at first, but since we are [caching each one of these requests](https://github.com/DoSomething/gambit-campaigns/blob/master/lib/phoenix.js#L121) in Redis, once the cache is warmed up in one request, all other calls will quickly return the cached campaigns.

Reply:
Hmm, well I can try deploying to staging... if it would be ok then this wouldn't be blocked!

Reply:
Yea, let's test performance on staging before unblocking. I'm optimistic the cache should make it bearable.

Reply:
Testing locally -- when I flush redis cache and `GET /index` with `NODE_ENV` set to `production`, I do get an Internal Server Error that I'm trying to track down. The subsequent `GET /campaigns` request is successful, as some campaigns are returned from cache.